docs/packages/airbnb/README.md

```diff
@@ -259,7 +259,7 @@ const { Neutrino } = require('neutrino');
 // Even if using .neutrinorc.js, you must specify it when using
 // the API
 module.exports = Neutrino()
-  .use('.neutrinorc.js')
+  .use(__dirname + '/.neutrinorc.js')
```

Comment:
I think the more correct solution here would be to pass the correct working directory to Neutrino so it can resolve anything else from the `root` dir:

```js
module.exports = Neutrino({ root: __dirname })
  .use('.neutrinorc.js')
  .call('eslintrc');
```

Reply:
Cool, I didn't know update that.  I'll update this PR.

Reply:
I tried this. In VSCode I get again
```
[Error - 12:07:54] Cannot read config file: d:\Work\Projects_Fabrica\frontend-js-snake\.eslintrc.js Error: Cannot find module 'C:\Program Files (x86)\Microsoft VS Code\package.json'
```